tools/pathwaymatcher/pathwaymatcher.xml

```diff
@@ -0,0 +1,332 @@
+<tool id="pathwaymatcher" name="Pathway Matcher" version="1.7">
+    <description>
+        PathwayMatcher is a software tool writen in Java to search for pathways related to a list of proteins in Reactome.
```

Comment:
I would remove the information about Java ... and just tell the user what the tool is doing.